paths.tex

```diff
@@ -13,48 +15,118 @@
+  connect $v_i$ to $v_j$ (with the possibility of $v_i = v_j$) between the time
+  $[t_0, t_f]$ with weight $w$. That is, all edges are to be treated as directed
+  edges with some weight and time-window (possibly instantaneous) in which they
+  are \textit{active}.
+\end{defn}
+
+With this generalized definition of a temporal network, we will begin to explain
+some common, specialized descriptors of these networks:
+
+\begin{defn}
+  We will describe a temporal graph as \textbf{undirected} $\forall e \in E$
+  if there exists $e' \in E$ such that $e = (v_i, v_j, t_0, t_f,w)$ and $e' =
+  (v_j, v_i, t_0, t_f, w)$.\footnote{As we consider the real-life applications
+  of these temporal networks, it may be convenient to define an edge as
+  a 6-tuple with the last component $l$ being a unique label or edge
+  differentiator. This would allow for the following scenario: if author
+  \texttt{A} and \texttt{B} collaborate in window $[t_0, t_f]$ on multiple
+  papers with the same weight, we could differentiate two edges connecting the
+  two authors instead of condensing the two paper collaborations into one edge.
+  In many cases, it may not make sense to do this, but it is valuable to have
+  an umbrella definition that encompasses every type of graph we could create.}
+
+  Informally, if for every directed edge in the graph, there exists a
+  corresponding edge in the opposite direction connecting the same two
+  vertices and labeled with the exact same $t_i, t_f,$ and $w$, we will consider
+  the temporal graph to be undirected.
+\end{defn}
+
+\begin{defn}
+  We will describe a temporal graph as \textbf{unweighted} if every edge in $E$
+  has equal weight $w$.
+\end{defn}
+
+\begin{defn}
+  \textbf{Edge persistence} is the amount of time for which an edge is present.
+  More formally, in a graph $(V,E)$ the persistence of an edge $(u,v,t_1,t_2,
+  w)$ is defined as $t_2 - t_1$ or simply $\delta t$.
+\end{defn}
+
+\begin{defn}
+  An edge $e \in E$ is said to be \textbf{infinitely persistent}
+  if $t_f = \infty$.
+\end{defn}
 
 \begin{defn}
-  An \textbf{edge-varying temporal network} is a tuple $G \in V \times E$ where
-  the set $V$ is the set of vertices, and $E \in V^2 \times T^2$, where $V$ is
-  the set of vertices and $T = [0,\infty)$.  The first vertex represents the
-  A \textbf{vertex-varying temporal network} is a tuple $G \in V \times E$ where
-  the set $V$ is the set of vertices -- each labelled with a start and end time,
-  and $E \in V^2$ , a pair of vertices.
-  When the vertex pair within the edges is ordered, call the graph
-  \textbf{directed}; otherwise call it \textbf{undirected}.
+  A \textbf{instantaneous edge} or \textbf{contact edge} is any edge $e \in E$
+  where $t_0 = t_f$.
 \end{defn}
 
-Specifically we will analyze the effect of these definitions on the coauthorship
-and citation networks.
+\begin{defn}
+  \textbf{Interval networks} or \textbf{windowed networks} shall be defined as
```

Comment:
We need to be careful with terminology here.  An interval graph is a static structure that is often used in a very different way than we are describing (its something I've been intending to bring up but haven't since it hasn't been pressing, might be useful for our definitions of contemporaneity). Since you above said that network and graph are interchangable, we'll get into trouble when we want to introduce these interval graphs.